src/test/regress/citus_tests/config.py

```diff
@@ -122,6 +123,8 @@ def post_init(self):
         self.output_file = os.path.join(self.datadir, "run.out")
         if self.worker_amount > 0:
             self.chosen_random_worker_port = self.random_worker_port()
+        if self.create_user is None:
```

Comment:
I guess if we change the variable names (as per other comment), it'd be easier to follow. Still, could use 1 line comment